virtualization/windowscontainers/kubernetes/getting-started-kubernetes-windows.md

```diff
@@ -1,152 +1,192 @@
+[Environment]::SetEnvironmentVariable("HTTPS_PROXY", "http://proxy.example.com:443/", [EnvironmentVariableTarget]::Machine)
+```
+
+There is a collection of scripts on [this Microsoft repository](https://github.com/Microsoft/SDN) that will help us join this node to the cluster. You can download the ZIP file directly [here](https://github.com/Microsoft/SDN/archive/master.zip). The only thing we need is the `Kubernetes/windows` folder, the contents of which should be moved to `C:\k\`:
+
+```powershell
+wget https://github.com/Microsoft/SDN/archive/master.zip -o master.zip
+Expand-Archive master.zip -DestinationPath master
+mkdir C:/k/
+mv master/SDN-master/Kubernetes/windows/* C:/k/
+rm -recurse -force master,master.zip
+```
+
+Copy the certificate file [identified earlier](#preparing-the-linux-master) to this new `C:\k` directory.
+
+
+### Creating the "Pause" Image ###
+Now that `docker` is installed, we need to prepare a "pause" image that's used by Kubernetes to prepare the infrastructure pods.
+
+```powershell
+docker pull microsoft/windowsservercore:1709
+docker tag microsoft/windowsservercore:1709 microsoft/windowsservercore:latest
+cd C:/k/
+docker build -t kubeletwin/pause .
+```
+
+> [!Note]  
+> We tag it as the `:latest` because the sample service we will be deploying later depends on it, though this may not actually _be_ the latest Windows Server Core image available. It's important to be careful of conflicting container images; not having the expected tag can cause a `docker pull` of an incompatible container image, causing [deployment problems](./common-problems.md#when-deploying-docker-containers-keep-restarting). 
+
+
+### Downloading Binaries ###
+In the meantime while the `pull` occurs, download the following client-side binaries from Kubernetes:
+
+  - `kubectl.exe`
+  - `kubelet.exe`
+  - `kube-proxy.exe`
+
+You can download these from the links in the `CHANGELOG.md` file of the latest 1.9 release. As of this writing, that is [1.9.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.9.1), and the Windows binaries are [here](https://storage.googleapis.com/kubernetes-release/release/v1.9.1/kubernetes-node-windows-amd64.tar.gz). Use a tool like [7-Zip](http://www.7-zip.org/) to extract the archive and place the binaries in `C:\k\`.
+
+In order to make kubectl command available outside of `C:\k\` directory, modify PATH environment variable:
+```powershell
+$env:Path = $env:Path + ";C:\k"
+```
+If you would like to make this change permanent, modify the variable in machine target:
+```powershell
+[Environment]::SetEnvironmentVariable("Path", $env:Path + ";C:\k", [EnvironmentVariableTarget]::Machine)
+```
+
+> [!Warning]  
+> As of this writing, `kube-proxy.exe` requires a pending Kubernetes [pull request](https://github.com/kubernetes/kubernetes/pull/56529) to work properly. You may need to [build the binaries manually](./compiling-kubernetes-binaries.md) to work around this.
```

Comment:
@gkudra-msft - I had to do a manual merge here. I see that PR referenced is merged but not tagged with a version. Should we keep this note or is the kube-proxy fix in 1.9.1?

Reply:
The warning is no longer needed. My only concern is the massive diff size for what appears to be a small number of actual changes; was there a line-ending (CRLF <--> LF) change? Please just make sure it's consistent with the other pages! Otherwise I have one other _very_ small nitpick in another comment that you are free to ignore. :)